rules/manager.go

```diff
@@ -49,7 +45,7 @@ var (
 			Name:      "rule_evaluation_duration_seconds",
 			Help:      "The duration for a rule to execute.",
 		},
-		[]string{ruleTypeLabel},
+		[]string{"rule_type", "status"},
```

Comment:
The consensus is that you only do this on a latency metric if a significant level of "failure" is a completely normal and expected behaviour. That's not the case here.

Failures and latency should almost always be separate metrics. If you split out like this PR does people will only graph the success latency, and not understand why their latency sucks when there's lot of timeouts.